src/meltano/cli/interactive/config.py

```diff
@@ -299,17 +299,24 @@ def configure(self, name, index=None, last_index=None, show_set_prompt=True):
 
     def configure_all(self):
         """Configure all settings."""
+        numeric_choices = [idx for idx, _, _ in self.setting_choices]
+        if not numeric_choices:
+            click.secho("There are no settings to configure.", fg="yellow")
```

Comment:
@WillDaSilva - Let me see if there is any docs link we can provide here for guidance.

Reply:
This is the link we provide on the Hub for defining/overriding settings:

- https://docs.meltano.com/guide/configuration#overriding-discoverable-plugin-properties

Here are two other related links but both lack context on exactly the steps to resolve:

- https://docs.meltano.com/guide/configuration#custom-settings
- https://docs.meltano.com/reference/plugin-definition-syntax#settings

Reply:
Given that this is a pretty common issue and I don't feel great about any of these resources being very targeted or helpful, I'm leaning towards a new "No Settings Defined" section (or similar) and a shortcut URL: https://melta.no#no-settings-defined.

cc @tayloramurphy